nginx/default.conf

```
@@ -13,14 +15,66 @@ server {
 
     server_name ${APPLICATION_URL};
 
-    location /static/ {
-        root /var/html/;
+    # Обходим падение swag если сервис frontend не поднят в той же сети
+    set $frontend "http://frontend";
+
+    location /api/(docs|redoc|openapi.json) {
+        proxy_pass http://backend:8000/;
+    }
+
+    location /api {
+        return 302 /api/;
+    }
+
+    location /api/ {
+        proxy_pass http://backend:8000/;
     }
 
     location / {
+        root /var/html/;
+        proxy_pass $frontend;

```

Так работать не будет?
```suggestion
        proxy_pass http://frontend;
```